Ghidra/Processors/MIPS/data/languages/mips32Instructions.sinc

```diff
@@ -1644,7 +1644,14 @@ define pcodeop SYNC;
 	build RTsrc;
 	tmp:$(REGSIZE) = sext(simmed:2) + RTsrc;	
 	JXWritePC(tmp); 
-    call [pc];
+    goto [pc];
+}
+
+:jic RTsrc, simmed                                      is $(AMODE) & REL6=1 & prime=0x36 & jsub=0x00 & RTsrc & simmed & rt=0x1f {
```

Comment:
If simmed != 0 is this really a return? I would think we would want to restrict this to just the case that simmed=0, in which case I would rewrite it as:
`:jic RTsrc, immed  is $(AMODE) & REL6=1 &prime=0x36 & jsub=0x0 & RTsrc & simmed & rt=0x1f & immed=0`

This would also allow you to shortcut the use of tmp and just do JXWritePC(ra);

Reply:
Nice suggestion